### DISCUSSION TITLE
How to update kvssink parameters at run time ?

### DISCUSSION THREAD

**Discussion by umerm64** [awslabs/amazon-kinesis-video-streams-producer-sdk-cpp#1040] [Q&A]
I want to update kvssink parameters like stream-name, access-key, secret-key for a running pipeline.
The objective is to send stream to multiple kinesis accounts based on certain events.
I am trying to use `g_object_set` function to do this but right now I cant seem to do it.
Need help in this regard.

Thank you.

**Comment by disa6302** [comment]
Answered in the issue #1044 . Re-iterating here:

We do not support pipeline PAUSING at the moment. The parameters for kvssink need to be set before changing pipeline to PLAYING STATE. This means, you can do this only at the start. To update again, you would have to stop the pipeline, set the params and set it to PLAYING again.
